package.json

```diff
@@ -148,6 +148,7 @@
     "publish-release": "^1.6.1",
     "resin-lint": "^3.2.4",
     "rewire": "^3.0.2",
+    "simple-git": "^1.131.0",
```

Comment:
[simple-git](https://www.npmjs.com/package/simple-git) is a lightweight wrapper around executing `git`, conveniently parsing the output as objects and arrays. The `git` executable must still be available.